### PR TITLE
Add more Swift extensions

### PR DESCRIPTION
**Pull request by bryanluby**
This pr is mainly for adding more convenience Swift extension methods:

- Added `UIView` extensions for pinning to a superview and finding the first superview of a given type.
- Added `UIViewController` extensions for adding/removing child view controllers. Also embedding a view controller in a navigation controller.
- Added `UIStackView` extensions for adding/removing arranged subviews.
- Added `UIPageControl` extension to refresh the current page.
- Added `UIApplication` extension to dismiss the root view controller.
- Refactored to `ReuseIdentifiableView`.
- Bumped iOS deployment target to 11 and cleaned out deprecations.
- Cleaned up podspec.

@vokal/ios-developers review please

### CODE REVIEW

File: Pod/VOKEmailHelper/VOKEmailHelper.m [hunk 1]
```diff
@@ -75,27 +75,18 @@ + (void)showSimulatorMailIsBorked:(UIViewController *)presenter message:(NSStrin
...
-        //iOS 7 - Use UIAlertView
-        UIAlertView *alertView = [[UIAlertView alloc] initWithTitle:title
-                                                            message:messageString
-                                                           delegate:nil
-                                                  cancelButtonTitle:cancelButtonTitle
-                                                  otherButtonTitles:nil];
-        [alertView show];
-    }
+    
+    UIAlertController *simulatorSucksAlert = [UIAlertController alertControllerWithTitle:title
```
Comment (vmammanalupo) [inline Pod/VOKEmailHelper/VOKEmailHelper.m:79]: lmao

**Review by vmammanalupo — APPROVED** [review]
Good stuff

File: Pod/VOKEmailHelper/VOKEmailHelper.m (hunk 1 above)
Comment (designatednerd) [reply Pod/VOKEmailHelper/VOKEmailHelper.m:79]: ^ pretty sure this is my fault 😛

Comment (bryanluby) [reply Pod/VOKEmailHelper/VOKEmailHelper.m:79]: Perfectly named

Comment (brockboland) [reply Pod/VOKEmailHelper/VOKEmailHelper.m:79]: Simpler times, man

**Pull request merged**